### ISSUE TITLE
Change in readme regarding api key placement

### ISSUE THREAD

**Issue by aleks-tpom6oh** [antoniolg/Bandhook-Kotlin#19]
In gitignore there is an instruction:

```
# Api key
app/src/main/res/values/api_key.xml
```

I believe the readme should tell about it and ask to store api key there rather than in the config.xml file

**Comment by antoniolg** [comment]
Yeah, the README is wrong. Feel free to fix it if you want, or I'll do it in the next days.
